lib/service/checkpayments.go

```diff
@@ -0,0 +1,94 @@
+package service
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/getAlby/lndhub.go/db/models"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+)
+
+func (svc *LndhubService) CheckAllPendingOutgoingPayments(ctx context.Context) (err error) {
+	//check database for all pending payments
+	pendingPayments := []models.Invoice{}
+	//since this part is synchronously executed before the main server starts, we should not get into race conditions
+	err = svc.DB.NewSelect().Model(&pendingPayments).Where("state = 'initialized'").Where("type = 'outgoing'").Scan(ctx)
```

Comment:
should we add a time limit here? even though after this change we should not have old initialized ones, correct?